package.json

```diff
@@ -3,6 +3,10 @@
   "version": "8.6.0",
   "description": "Automated auditing, performance metrics, and best practices for the web.",
   "main": "./lighthouse-core/index.js",
+  "exports": {
+    ".": "./lighthouse-core/index.js",
+    "./renderer": "./dist/report/bundle.esm.js"
+  },
```

Comment:
When exports is defined, it means all exports are not allowed unless listed here. See the output from build-devtools:

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lighthouse-core/lib/i18n/i18n' is not defined by "exports" in /Users/runner/work/lighthouse/lighthouse/lighthouse/node_modules/lighthouse/package.json
```

Adding this line should fix it:

```
"./lighthouse-core/*": "./lighthouse-core/*"
```